pkg/component/delete/delete.go

```diff
@@ -61,15 +61,15 @@ func (do *DeleteComponentClient) ListClusterResourcesToDelete(componentName stri
 	return result, nil
 }
 
-func (do *DeleteComponentClient) DeleteResources(resources []unstructured.Unstructured) []unstructured.Unstructured {
+func (do *DeleteComponentClient) DeleteResources(resources []unstructured.Unstructured, wait bool) []unstructured.Unstructured {
```

Comment:
Maybe I missed it, but per the acceptance criteria in #5565, I didn't see any sensible timeout defined when `wait` is `true`:

> there should be some sensible timeout so it doesn't end in an infinite loop.

Reply:
Oh wait, I just saw a timeout defined in `WaitDynamicResourceDeleted`. So please ignore this comment :)